Dockerfile

```diff
@@ -1,10 +1,65 @@
-FROM php:7
-RUN apt-get update -y && apt-get install -y openssl zip unzip git
+FROM php:7.2-fpm
```

Comment:
This file basically tells docker what to do with the docker image when it's being built.

The `docker-compose build` happens automatically first time you run `docker-compose up`. If you make changes to this file `Dockerfile` you will probably need to run that command manually to have the changes reflected